dev/com.ibm.websphere.appserver.features/visibility/public/mongodb-2.0/resources/l10n/com.ibm.websphere.appserver.mongodb-2.0.properties

```diff
@@ -14,6 +14,5 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature enables the usage of the MongoDB Java Driver and allows DB instances to be \
- configured in the server configuration, injected into managed components such as EJBs, and accessed \
- through JNDI. Applications interact with these DB instances via the MongoDB APIs
+description=This feature has been stabilized. \
```

Comment:
Can we append this information to the existing feature description? We should still have the description of what this feature does, even if it is stabilized and there is a more strategic alternative.

Reply:
We could, I was just trying to make it harder to use since we want people to move off.  I'll prepend this info so it comes first.